### PR TITLE
[mergebot] Set mergebot to be configured via config

### PR DESCRIPTION
**Pull request by clee2000**
For repos that want to merge via `@pytorchbot merge`, the repo should put `mergebot: True` in their `pytorch-probot.yml`

In preparation for letting torchao trigger their own merges through bot

Risks: 
If we can't query for the config for some reason, it will not merge

todo before this gets merged:
Add `mergebot: True` in pytorch's config file

### CODE REVIEW

File: torchci/lib/bot/pytorchBot.ts [hunk 1]
```diff
@@ -1,8 +1,11 @@
 import { Probot } from "probot";
 import { getInputArgs } from "./cliParser";
 import PytorchBotHandler from "./pytorchBotHandler";
+import { CachedConfigTracker } from "./utils";
 
 function pytorchBot(app: Probot): void {
+  const cachedConfigTracker = new CachedConfigTracker(app);
```
Comment (ZainRizvi) [inline torchci/lib/bot/pytorchBot.ts:7]: any idea how long the cache is persisted?

**Review by ZainRizvi — APPROVED** [review]
🚢

**Review by huydhn — APPROVED** [review]
LGTM! Pending your change on PT bot config

File: torchci/lib/bot/pytorchBot.ts (hunk 1 above)
Comment (clee2000) [reply torchci/lib/bot/pytorchBot.ts:7]: [I think the cache is not very good](https://github.com/pytorch/test-infra/pull/5098)

Comment (malfet) [reply torchci/lib/bot/pytorchBot.ts:7]: It's not persisted at all, but repeated calls to config methods during execution of the command would not need to make new network calls to fetch the same info.

**Review by malfet — APPROVED** [review]

**Pull request merged**